### PR TITLE
Fix custom dependency path

### PR DESCRIPTION
**Pull request by lbovet**

### CODE REVIEW

File: src/generate-schema.js [hunk 1]
```diff
@@ -64,8 +64,11 @@ module.exports = {
                 var schema = schemas[name];
                 //thank you MS for messing around with filenames!
                 var filename = normPath(schema.extra.filename);
-                var source = filename.toLowerCase().startsWith(normPath(deps).toLowerCase())
-                    ? (relDeps + '/ts/' + path.dirname(filename.substring(filename.lastIndexOf('-api-dependencies/ts/') + 21, filename.lastIndexOf('/ts')+3)) + '/json-schema-v4/') : '';
+                var relativeDir = params.isCustomDependencyPath() ?
+                    (relDeps + '/json-schema-v4' + path.dirname(filename.substring(deps.length + 3)) + '/') :
+                    (relDeps + '/ts/' + path.dirname(filename.substring(filename.lastIndexOf('-api-dependencies/ts/') + 21, filename.lastIndexOf('/ts')+3)) + '/json-schema-v4/');
+                relativeDir =
```
Comment (hiddenalpha) [inline src/generate-schema.js:70]: `relativeDir = `

What is this line for (line 70)? Makes no sense to me.

File: src/params.js [hunk 2]
```diff
@@ -11,6 +11,8 @@ var models = noSlash(env.models || 'ts');
 
 var packageJson = fs.existsSync('./package.json') ? JSON.parse(fs.readFileSync('./package.json')) : {};
 
+const defaultDependencyPath = 'node_modules/-api-dependencies';
```
Comment (hiddenalpha) [inline src/params.js:14]: That path looks strange with this leading dash in the 2nd segment. Is this by intent?

File: src/generate-schema.js (hunk 1 above)
Comment (lbovet) [reply src/generate-schema.js:70]: Yep, that was stuff that I accidentally committed... argh.
It is fixed now on develop.

File: src/params.js (hunk 2 above)
Comment (lbovet) [reply src/params.js:14]: It was designed this way from the beginning. Maybe it was to avoid clashes with real modules.

**Pull request merged**